docs/contributor/ADR/002-command-structure-standards.md

```diff
@@ -0,0 +1,277 @@
+  kyma module add my-namespace/my-module-template-name --default-cr --auto-approve
+```
+
+The help will displays:
+
+```bash
+kyma module add --help
+
+Use this command to add a module.
+
+Usage:
+  kyma module add <module> [flags]
+
+Examples:
+  # Add a Kyma module with the default CR
+  kyma module add kyma-module --default-cr
+
+  # Add a Kyma module with a custom CR from a file
+  kyma module add kyma-module --cr-path ./kyma-module-cr.yaml
+
+  ## Add a community module with a default CR and auto-approve the SLA
+  #  passed argument must be in the format <namespace>/<module-template-name>
+  kyma module add my-namespace/my-module-template-name --default-cr --auto-approve
+```
+
+### Examples details
+
+Every line of the examples must start with double spaces to display them correctly in the terminal.
+
+All examples for one command must be separated with an empty line and have their own description starting from the `#` symbol. If the example description is longer than one line, then the first line should start with the `##` prefix, and then every next line should start with `#` and two spaces after.
+
+Examples should reflect real use cases, so if possible, they should use real data as arguments and flags if possible. If not, then use fiction one representing real data, like `my-module`, `my-resource`, `my-something`.
+
+## Aliases
+
+The alias is an array table in the `cobra.Command{}.Aliases`, and there are no specific requirements about command aliases. The good idea is always to use this functionality to provide a shorthand of the command (`del` for `delete` command), or a word form that can help with avoiding small typos (`modules` for `module` command).
+
+### Aliases in practice
+
+```go
+cmd := &cobra.Command{
+  Use:   "delete <module> [flags]",
+  Aliases: []string{"del"},
+}
+```
+
+## Errors&Hints
+
+Errors were proposed in the [ADR 001](001-error-output-format.md) proposal and implemented quite after that. This functionality allows users to understand what is going on on three levels of abstraction. The general message called `Error` should contain the last, user-understandable operation that fails. The second thing is `Error Details` that can contain an internal error message generated from the library or called server. The `Hints` section is something that should help users figure out how the problem can be fixed. Every hint should be in one of two formats:
+
+* `to <what>, <do>` - format used to describe possible optional configurations that may be used or misconfigured
+* `make sure <what>` - format used to describe the required configuration user may have misconfigured
+
+The cli is designed to always return `clierror.Error` instead of pure `error`. Both errors are not compatible with each other, and to avoid user confusion, we should not use the `cobra.Command{}.RunE` and instead of that use the `cobra.Command{}.Run` and check the error manually inside of it:
```

Comment:
```suggestion
The CLI is designed to always return `clierror.Error` instead of pure `error`. Both errors are not compatible with each other, and to avoid user confusion, we should not use the `cobra.Command{}.RunE` and instead of that use the `cobra.Command{}.Run` and check the error manually inside of it:
```